src/favicon_image/svg.rs

```diff
@@ -0,0 +1,52 @@
+//! Svg operations for favicon images
+
+use image::{DynamicImage, RgbaImage};
+use lazy_static::lazy_static;
+use resvg::{
+    tiny_skia,
+    usvg::{self, fontdb, Options, Size, TreeParsing, TreeTextToPath},
+    Tree,
+};
+
+// Load fonts once
+lazy_static! {
+    static ref FONT_DB: fontdb::Database = {
+        let mut db = fontdb::Database::new();
+
+        // Load system fonts if available
+        db.load_system_fonts();
+
+        // Load any fonts in the current directory
+        if let Ok(pwd_path) = std::env::current_dir() {
+            db.load_fonts_dir(pwd_path);
```

Comment:
Should we instead provide a command line option to specify a font file? Right now it will only work with files called `Arial.ttf`.

Reply:
Fwiw right now it will load all the font files it finds but then just use the one with a font family name that matches Arial. I think we'd be better off supporting a flag to choose a different font for the fallback but maybe we plan that for a future version